### PR TITLE
Use Composer autoloader instead of deprecated Psr4ClassLoader

### PR DESCRIPTION
**Pull request by ossinkine**

### CODE REVIEW

File: FervoEnumBundle.php [hunk 1]
```diff
@@ -35,26 +35,18 @@ public function boot()
...
-            $this->autoloader = null;
-        }
-    }
-
-    private function registerAutoloader($container)
+    private function registerAutoloader(ContainerInterface $container)
     {
-        if (!$this->autoloader) {
+        if (!$this->autoloaderRegistered) {
+            $projectDir = $container->getParameter('kernel.project_dir');
```
Comment (magnusnordlander) [inline FervoEnumBundle.php]: `kernel.project_dir` only appeared in Symfony 3.3. We need to emulate it using `kernel.root_dir` for earlier versions.

**Review by magnusnordlander — CHANGES_REQUESTED** [review]
Thanks again for your pull request! This is great, I never even considered using the Composer autoloader.

There is however the slight problem of compatibility with Symfony 2.8 and <3.3, which would have to be fixed before merging, but again, this is great.

File: FervoEnumBundle.php (hunk 1 above)
Comment (ossinkine) [reply FervoEnumBundle.php]: What do you think, should I use if statement like
```php
if (Kernel::VERSION < 30000) {
    $projectDir = $container->getParameter('kernel.root_dir').'/..';
} else {
    $projectDir = $container->getParameter('kernel.project_dir');
}
```
to simplify further search when we will drop Symfony 2.8 support or just always use `$projectDir = $container->getParameter('kernel.root_dir').'/..';`?

Comment (magnusnordlander) [reply FervoEnumBundle.php]: No, an if statement that we can drop when 2.8 support is dropped is definitely the best.

Comment (ossinkine) [reply FervoEnumBundle.php]: Updated, please review

**Comment by magnusnordlander** [comment]
Awesome, thanks!

**Pull request merged**